scripts/hydrogen_bond_propensity/hydrogen_bond_propensity_report.py

```diff
@@ -20,6 +21,7 @@
 import sys
 import subprocess
 import csv
+import string
```

Comment:
I am not 100% sure where this being used 🤔

Reply:
In case want to label each component